pgroll-integration.mdx

```diff
@@ -78,10 +79,24 @@ Integrating pgroll within our web application and developer packages has posed s
 - **Detecting when a migration is complete**
   - pgroll does not allow for referencing items created in previous operations in subsequent operations within the same migration. This means that running an `alter_column` operation after an `add_column` operation, or running a `rename_constraint` operation (requires a table name) after a `rename_table` operation within the same migration request is not currently possible. This is due to the fact that the first operation in the list is guaranteed to compete before the second operation completes, but the first operation is not guaranteed to complete before the second operation starts. Since the backend provides an asynchronous `apply` endpoint which responds with a `jobId`, we decided that the frontend would use the returned `jobId` as input to a second new endpoint called `status` which would return the state of that migration. The frontend polls the status endpoint until the migration in question succeedes or fails before moving forward with dependent migrations.
 
-## Plans for GA
+## Path to General Availability
+
+While this is a very exciting release for us, there's still a bit more we need to do before getting this integration over the finish line.
+
+The magic ✨ of pgroll is that it provides the ability to write to two schemas at the same time while your application migrates over to the new schema. For our initial release, this functionality didn't make the cut and instead we wanted to focus on unlocking new types of migrations for our users. Before we GA this integration and approach to schema migrations, we'll provide support for multiple schemas.
```

Comment:
The name schema is a bit overloaded because postgres also uses schema to identify namespaces (ie `public`). This could make things more clear, but feel free to ignore this comment

```suggestion
The magic ✨ of pgroll is that it provides the ability to write to two versions of the schema at the same time while your application migrates over to the new schema. For our initial release, this functionality didn't make the cut and instead we wanted to focus on unlocking new types of migrations for our users. Before we GA this integration and approach to schema migrations, we'll provide support for multiple schemas.
```